config.go

```diff
@@ -686,6 +688,17 @@ func LoadConfig(interceptor signal.Interceptor) (*Config, error) {
 		os.Exit(0)
 	}
 
+	if preCfg.GenerateManPage {
+		fileParser := flags.NewParser(&preCfg, flags.Default)
```

Comment:
Couldn't this also be done in the `lncli generatemanpage` command? Doesn't look like we need the actual content of the `preCfg`, so we can just use the default config.

Reply:
Hey @guggero ! could you please elaborate more on this?

Reply:
Thanks, I figured it out while working on the coin-selection-strategy option issue. I just needed more time to get hands-on with the codebase!